src/Raven.Server/Documents/Indexes/IndexDefinitionBaseServerSide.cs

```diff
@@ -176,12 +176,13 @@ public static class IndexVersion
 
             public const long PhraseQuerySupportInCoraxIndexes = 60_002;
             public const long StoreOnlySupportInCoraxIndexes = 60_003; // RavenDB-22369
-            public const long JavaScriptProperlyHandleDynamicFieldsIndexFields = 60_004; //RavenDB-22363
+            public const long JavaScriptProperlyHandleDynamicFieldsIndexFields = 60_004; // RavenDB-22363
+            public const long LoadDocumentWithDynamicCollectionNameShouldThrow = 60_005; // RavenDB-22359
```

Comment:
This is on 6.1 version so I believe we need to bump it to 61_000. We might want to use 60_005 on 6.0 branch

Reply:
@ppekrol @maciejaszyk ?

Reply:
Agree. We've similar thing done for `TimeTicksSupportInJavaScriptIndexes`

Reply:
Fixed